uwblueprint/extend-a-family
# Change invited admin user password on first login

## Notion ticket link

[Change Password on First Login for Invited User](https://www.notion.so/uwblueprintexecs/Change-Password-on-First-Login-for-Invited-User-7eab8e395ae44fb9a1c3c109a05443ad)



## Implementation description
* Used Firebase's REST API to change password using user's access token.
* Split logic for change password and updating user status into two separate endpoints.



## Steps to test
1. Post the invite admin endpoint in the backend using Postman and a manually obtained admin access token to invite a new admin user
2. Verify and login using the new credentials
3. Choose a new password and check that your user is now `Active`.



## What should reviewers focus on?
* The process flow and potential security issues.


## Checklist
